halo-dev/halo
# Fix the problem of unable to start application through Docker image

#### What type of PR is this?

/kind bug
/area core

#### What this PR does / why we need it:

See https://github.com/halo-dev/halo/pull/4850#issuecomment-1812068735 for more.

Test steps:

```bash
make -C console build
./gradlew clean build -x check
docker build -t halo:not-find-main-class .
docker run -it --rm -p 8090:8090 -v ~/.halo2:/root/.halo2 halo:not-find-main-class
```

#### Does this PR introduce a user-facing change?

```release-note
None
```
